CMakeLists.txt

```diff
@@ -192,8 +192,13 @@ set(fbgemm_targets)
 ################################################################################
 
 if(CMAKE_SYSTEM_PROCESSOR MATCHES "x86_64|AMD64|i386|i686")
-  if(NOT COMPILER_SUPPORTS_AVX2)
-    message(FATAL_ERROR "The current compiler does not support building AVX2 code")
+  set(AVX2_flags)
+  if(CXX_AVX2_FOUND)
+    if(MSVC)
+      set(AVX2_flags -mavx2)
+    else()
+      set(AVX2_flags -mavx2)
```

Comment:
What's the difference between the two cases here?

Reply:
Typo